pkg/cmd/application/create/app_create.go

```diff
@@ -58,10 +61,15 @@ NOTE:
 				opts.LocalPath = "./" + opts.Name
 			}
 			opts.Streams = userio.NewIOStreamsWithInteractive(
-				cmd.InOrStdin(),
-				cmd.OutOrStdout(),
+				os.Stdin,
```

Comment:
Is there a reason for replacement? Previous implementation allowed to mock stdin/stdout

Reply:
using the cobra versions wasn't picking up any of the windowResize signals, the tests are setting up the streams with io.Buffers still

Reply:
Hm, weird.
Should we replace it in every cmd then?

Reply:
Just curious, have you tried to find the root cause?
I just quickly looked at the source code and it looks like it should return `os.Stdin/os.Stdout` in case it's not overwritten